### DISCUSSION TITLE
ETH DRIVER    whether the  fpga-axi-eth.c driver works on 10Gbase-r network?

### DISCUSSION THREAD

**Discussion by zhou-he** [eugene-tarassov/vivado-risc-v#258] [Q&A] [answered]
hello ,I want to know that whether the  fpga-axi-eth.c driver works on 10Gbase-r network? If not, how can it be modified to drive 10gbase-r Ethernet networks.   Looking forward to receiving your recovery, thank you very much!

**Comment by eugene-tarassov** [comment, marked as answer]
The driver does not depend on the link type or speed. All it does is moving data between memory and NIC using DMA. It should work with 10Gbase-r.

**Reply by zhou-he** [reply]
I modified 10gbase-r Ethernet to the chip of xcku060 according to the routine of vcu1525, and modified GTY to GTH, but it did not work properly after being put on the board. I do not know what the problem is, do you have any suggestions?

**Reply by zhou-he** [reply]
Can the 10gbase-r Ethernet in the project vcu525 u200 u250 you provided run linux system correctly? Have you tested it on the board? I have been stuck in this place for many days. I hope to receive your reply. Thank you very much!

**Reply by eugene-tarassov** [reply]
I run daily automated regression tests for vcu1525 - no issues.
